src/main/resources/static/admin/styles.css

```
@@ -255,6 +255,202 @@ button.secondary {
+	align-items: center;
+	padding: 0.14rem 0.46rem;
+	border-radius: 999px;
+	background: rgba(13, 125, 84, 0.16);
+	color: #124e39;
+	font-weight: 700;
+}
+
+.table-pre {
+	margin: 0;
+	white-space: pre-wrap;
+	word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)